### PR TITLE
Service/PCTL: convert to module, add services, stub

### PR DESCRIPTION
**Pull request by mailwl**
PCTL::CreateServiceWithoutInitialize and IParentalControlService::Initialize, required by Kirby Star Allies

### CODE REVIEW

File: src/core/hle/service/pctl/module.cpp [hunk 1]
```diff
@@ -108,20 +109,40 @@ class IParentalControlService final : public ServiceFramework<IParentalControlSe
         };
         RegisterHandlers(functions);
     }
+
+private:
+    void Initialize(Kernel::HLERequestContext& ctx) {
+        LOG_WARNING(Service_PCTL, "(STUBBED) called");
+        IPC::RequestParser rp{ctx};
```
Comment (bunnei) [inline src/core/hle/service/pctl/module.cpp]: This isn't used

File: src/core/hle/service/pctl/module.cpp [hunk 2]
```diff
@@ -108,20 +109,40 @@ class IParentalControlService final : public ServiceFramework<IParentalControlSe
...
+
+        IPC::ResponseBuilder rb = rp.MakeBuilder(2, 0, 0);
```
Comment (bunnei) [inline src/core/hle/service/pctl/module.cpp]: This can just be IPC::ResponseBuildser rb{ctx, ..}

**Comment by Hexagon12** [comment]
I don't know if this is related, but Skyrim calls pctl:CreateService, throws some unmapped reads and the emulated program breaks execution. I suggest looking into this if you have the game and you think it's related.

```[  11.631831] Service.PCTL <Debug> core\hle\service\pctl\module.cpp:CreateService:127: called
[  11.633667] HW.Memory <Error> core\memory.cpp:Read:172: unmapped Read64 @ 0x000000C0
[  11.633968] HW.Memory <Error> core\memory.cpp:Read:172: unmapped Read8 @ 0x00000060
[  11.634034] HW.Memory <Error> core\memory.cpp:Read:172: unmapped Read64 @ 0x00000068
[  11.634489] HW.Memory <Error> core\memory.cpp:Read:172: unmapped Read8 @ 0x00000060
[  11.634819] HW.Memory <Error> core\memory.cpp:Read:172: unmapped Read64 @ 0x00000020
[  11.635118] HW.Memory <Error> core\memory.cpp:Read:172: unmapped Read64 @ 0x00000028
[  11.637265] Service.AM <Warning> core\hle\service\am\am.cpp:SetTerminateResult:536: (STUBBED) called, result=0x000002A2
[  11.638387] Debug.Emulated <Critical> core\hle\kernel\svc.cpp:Break:276: Emulated program broke execution!
[  11.638714] Debug <Critical> core\hle\kernel\svc.cpp:operator ():277: Assertion Failed!```

**Comment by mailwl** [comment]
i will look, but most probably not related, at least Kirby pass 2 functions of pctl till unimplemented set:GetLanguageCodes2

**Comment by lioncash** [comment]
`NGLOG_` should be used instead of the old `LOG_` macros

**Comment by bunnei** [comment]
lgtm after merge conflicts are addressed

**Pull request merged**